File: C++ WorldSimulatorGame/POproj1/Trawa.cpp
#include "Trawa.h"


Trawa::Trawa(Punkt polozenie, Swiat* swiat)
	:Roslina(0, 0, 'T', 0, true, Organizmy::trawa, polozenie, swiat) {
	this->sila = 0;
	this->inicjatywa = 0;
	this->znak = 'T';
	this->wiek = 0;
}
string Trawa::getNazwaOrganizmu() {
	return "Trawa";
}

File: Python WorldSimulatorGame/Trawa.py
from Roslina import Roslina


class Trawa(Roslina):

    def __init__(self, polozenie, swiat):
        super(Trawa, self).__init__(0, 0, 'T', 0, True, polozenie, swiat)

    def getNazwaOrganizmu(self):
        return "Trawa"